evap/static/css/evap.scss

```diff
@@ -1033,6 +1033,10 @@ a.no-underline:hover {
     text-decoration: none;
 }
 
+.semester-short-name {
+    word-spacing: 30000px;
+}
```

Comment:
what does this do?

Reply:
...it makes the lines *always* break there, right?

in any case, please add a comment along those lines.

Reply:
Yes, for consistency with different semester name lengths between ST and WT. Will do.